apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -223,6 +226,16 @@ private async Task<int> ExecuteDmlAsync(CancellationToken cancellationToken)
                 return checked((int) count);
             }
 
+            private async Task<SpannerDataReader> ExecuteDmlReturnAsync(CancellationToken cancellationToken)
```

Comment:
I'd rename this to `ExecuteDmlReaderAsync` for the same reasons as on ISpannerTransaction.